probatus/sample_similarity/resemblance_model.py

```diff
@@ -414,28 +423,32 @@ class SHAPImportanceResemblance(BaseResemblanceModel):
     <img src="../img/sample_similarity_shap_summary.png" width="320" />
     """
 
-    def __init__(self, model, iterations=100, test_prc=0.25, n_jobs=1, random_state=42):
+    def __init__(self, model, test_prc=0.25, n_jobs=1, verbose=0, random_state=42):
```

Comment:
not familiar enough with code base, but are you sure you can remove this parameter?

Reply:
Yes, this parameter is not used in any way in ShapResemblance, it is only used in permutation one.